pkg/broker/broker.go

```diff
@@ -722,6 +724,10 @@ func (a AnsibleBroker) Deprovision(
 		log.Info("Synchronous deprovision in progress")
 		_, err = apb.Deprovision(&instance)
 		if err != nil {
+			if a.brokerConfig.ForceDelete {
+				log.Infof("Deprovision failed. Attempting to clean up related resources. User should ensure clean up is successful")
+				cleanupDeprovision(&instance, a.dao)
```

Comment:
we're ignoring the err returned by `cleanupDeprovision` at a minimum we need to log it so we can figure out what was really going on.

Reply:
We do ignore the error from `cleanupDeprovision` but that function does take care of logging the error. Also we return the original error to the caller. Do you think this is sufficient?